base/services/w32time/w32time.c

```diff
@@ -149,8 +151,8 @@ GetIntervalSetting(VOID)
         RegCloseKey(hKey);
     }
 
-    if (lRet != ERROR_SUCCESS)
-        return 0;
+    if (lRet != ERROR_SUCCESS || dwData < 120)
+        return 9 * 60 * 60; // 9 hours, because Windows uses 9 hrs, 6 mins and 8 seconds by default.
```

Comment:
[RFC 5905](https://tools.ietf.org/html/rfc5905#page-21) suggests a default of 2ˆ10 (~17 minutes).

Reply:
Thanks but it may be too long.

Reply:
This is the max value, not the default. What I meant is that we should probably fallback to the default when it's above MAXPOLL.